mmv1/products/apigee/TargetServer.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'TargetServer'
+base_url: '{{env_id}}/targetservers'
+create_url: '{{env_id}}/targetservers'
+delete_url: '{{env_id}}/targetservers/{{name}}'
+self_link: '{{env_id}}/targetservers/{{name}}'
+update_url: '{{env_id}}/targetservers/{{name}}'
+description: |
+  TargetServer configuration. TargetServers are used to decouple a proxy TargetEndpoint HTTPTargetConnections from concrete URLs for backend services.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Load balancing across backend servers': 'https://cloud.google.com/apigee/docs/api-platform/deploy/load-balancing-across-backend-servers'
+  api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.targetservers/create'
+import_format: ['{{env_id}}/targetservers/{{name}}', '{{env_id}}/{{name}}']
+skip_sweeper: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'apigee_target_server_test_basic'
+    primary_resource_id: 'apigee_target_server'
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+    skip_test: true
```

Comment:
Currently, [this example is shown in the documentation](https://github.com/modular-magician/terraform-provider-google/blob/f1bcc99e0201ae92261716ec3299f9aba810ffed/website/docs/r/apigee_target_server.html.markdown) with `"tf-test%{random_suffix}"` set as some arguments in resource blocks.

Could you please address this by adding a vars property to this Terraform::Examples block  ([see example of using vars here](https://github.com/GoogleCloudPlatform/magic-modules/blob/867d852a0251dcfc18663d3a65b5a425dc4ba1ad/mmv1/products/apigee/Envgroup.yaml#L50-L51)) and update the .../examples/apigee_target_server_test_basic.tf.erb file to include references to those vars ([example here](https://github.com/GoogleCloudPlatform/magic-modules/blob/867d852a0251dcfc18663d3a65b5a425dc4ba1ad/mmv1/templates/terraform/examples/apigee_environment_group_basic.tf.erb#L29))

Reply:
%{random_suffix} replaced with vars. Thanks.